src/routes/admin/profiles/edit.test.ts

```diff
@@ -350,6 +351,79 @@ test('test handleGenerateRandomMEBxPassword when the request body generateRandom
   expect(result).toEqual(expected)
 })
 
+describe('handleWifiConfigs tests', () => {
+  let newConfig: AMTConfiguration
+  let oldConfig: AMTConfiguration
+
+  const profileWifiConfigsDb = {
+    deleteProfileWifiConfigs: jest.fn().mockReturnValue(null),
+    createProfileWifiConfigs: jest.fn().mockReturnValue(null),
+    getProfileWifiConfigs: jest.fn().mockReturnValue(null)
+  }
+
+  beforeEach(() => {
+    newConfig = {
+      profileName: 'profileName1',
```

Comment:
technically on edit, profileName must be the same. I'm pretty sure that condition is caught elsewhere before the handling of the wifiprofiles, BUT I see that in the handling call, the delete uses the newconfig profile name instead of the old, and I believe if that case every actually happened, the delete would fail if the profile names didn't match. So I would suggest the delete calls should be changed to use oldConfig.profileName.